### DISCUSSION TITLE
Welcome to SugarCube v2 Discussions!

### DISCUSSION THREAD

**Discussion by tmedwards** [tmedwards/sugarcube-2#100] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hogart** [comment]
Hello everyone, I'm using Twine and SugarCube for about 5 years, during which I created [this collections](https://github.com/hogart/sugar-cube-utils) of SugarCube 2 related macros and utilities and [this browser extension](https://github.com/hogart/twine-enhancer) for Twine web. Sometimes I help people with the format, sometimes I myself need help.

**Reply by Jaydax1729** [reply]
I've been using Twine and Sugarcube for a year or so. My interest is in building interactive adventure stories. I've recently finished updating an adventure game first published for the BBC model B computer in 1982. Twine allows me to add graphics and sound to what was a text only game. Check out my first attempt of 'Castle of Riddles Revisited' at https://jaydax.co.uk/corr I've doubled the number of riddles, added many more objects to find and given you 50 ways to die.

How could Twine be improved? I'd like to see editor improvements. Currently, for a complex page I copy and paste the code into a text editor, edit and then paste back into Twine. Having search terms highlighted in the editor and things like <<link...>> <</link>> colour matched would make editing much easier. Something perhaps like this?
![2021-02-17_15-50-27-270](https://user-images.githubusercontent.com/50697568/108230081-4b474000-7138-11eb-8e8a-9eda29718dd4.png)

**Reply by hogart** [reply]
It's easier to develop big, complex stories using [twego](https://www.motoslave.net/tweego/) + [VS Code](https://code.visualstudio.com/) along with [Twee 3 Language Tools](https://marketplace.visualstudio.com/items?itemName=cyrusfirheir.twee3-language-tools). You get syntax highlight (like what you suggested) and checks for missing macros and arguments, and you can split you story into smaller, more manageable files. It's really game changer (pun intended):)
